x/cert/simulation/genesis.go

```diff
@@ -1,74 +1,21 @@
 package simulation
 
 import (
-	"math/rand"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-
 	"github.com/cosmos/cosmos-sdk/types/module"
-	"github.com/cosmos/cosmos-sdk/x/simulation"
 
 	"github.com/certikfoundation/shentu/x/cert/internal/types"
 )
 
 // RandomizedGenState creates a random genesis state for module simulation.
 func RandomizedGenState(simState *module.SimulationState) {
-	r := simState.Rand
 	gs := types.GenesisState{}
 
-	numOfCertifiers := r.Intn(100)
-	for i := 0; i < numOfCertifiers; i++ {
-		gs.Certifiers = append(gs.Certifiers, GenerateACertifier(r))
-	}
-
-	numOfValidators := r.Intn(10)
-	for i := 0; i < numOfValidators; i++ {
-		gs.Validators = append(gs.Validators, GenerateAValidator(r))
-	}
-
-	numOfPlatforms := r.Intn(10)
-	for i := 0; i < numOfPlatforms; i++ {
-		gs.Platforms = append(gs.Platforms, GenerateAPlatform(r))
-	}
-
-	numOfLibrary := r.Intn(20)
-	for i := 0; i < numOfLibrary; i++ {
-		gs.Libraries = append(gs.Libraries, GenerateALibrary(r))
+	for _, acc := range simState.Accounts {
+		if simState.Rand.Intn(100) < 10 {
+			certifier := types.NewCertifier(acc.Address, "", nil, "")
+			gs.Certifiers = append(gs.Certifiers, certifier)
```

Comment:
Is it difficult to set sim accounts for certifiers?

In gov's certifier voting round, there's a requirement on the percentage of certifiers who votes. It is very likely proposals will fail if some of the certifiers are not able to vote because they do not have sim accounts.

Or maybe we can remove original certifiers who do not have sim accounts after setting certifiers with sim accounts?

Reply:
After this modification, all certifiers will have valid accounts in simulation, we don't set meaningless addresses as certifiers anymore.

Reply:
Oh I misunderstood the previous implementation. Thanks!